tests/unit/test_tokens.py

```diff
@@ -99,15 +99,15 @@ def parametrize(cases):
 
 def _create_mock_session(config):
     mock_session = mock.Mock(spec=Session)
-    mock_session.get_config_variable.return_value = "test"
+    mock_session.get_config_variable.return_value = "default"
     mock_session.full_config = config
     return mock_session
 
 
 @parametrize(sso_provider_resolution_cases)
 def test_sso_token_provider_resolution(test_case):
     mock_session = _create_mock_session(test_case["config"])
-    resolver = SSOTokenProvider(mock_session)
+    resolver = SSOTokenProvider(mock_session, profile_name='test')
```

Comment:
Let's avoid updating existing test cases as they are covering that we can use the profile name from the session. Instead, let's get a new test case that is explicitly checking that we can override the profile name from what is on the session